eng/pipelines/libraries/stress/http.yml

```diff
@@ -57,6 +57,7 @@ jobs:
       export HTTPSTRESS_CLIENT_ARGS="$HTTPSTRESS_CLIENT_ARGS -http 3.0"
       export HTTPSTRESS_SERVER_ARGS="$HTTPSTRESS_SERVER_ARGS -http 3.0"
       docker-compose up --abort-on-container-exit --no-color
+    timeoutInMinutes: 35 # In case the HTTP/3.0 run hangs, we timeout shortly after the expected 30 minute run
```

Comment:
I'm adding this because I experienced mysterious HTTP/3.0 hangs on some runs:

https://dev.azure.com/dnceng/public/_build/results?buildId=1478084&view=logs&j=0da5d1d9-276d-5173-c4c4-9d4d4ed14fdb&t=1451f5f3-0108-5a08-5b92-e984b2a85bbd

Reply:
Do we know what happened? I see only blank screen (no output from the run)? Either way, we shouldn't hold this PR on it and investigate it separately.

Reply:
It stopped logging after [this](https://dev.azure.com/dnceng/public/_build/results?buildId=1481385&view=logs&j=0da5d1d9-276d-5173-c4c4-9d4d4ed14fdb&t=1451f5f3-0108-5a08-5b92-e984b2a85bbd&l=62) line, then that log disappeared for some reason after the whole job becoming cancelled. Unfortunately I wasn't able to reproduce this behavior locally.

I would be surprised if this is caused by my changes, we haven't seen working stress runs on the CI for a while.